smart-contracts/src/ibcMatchingTokens.ts

```diff
@@ -1,78 +1,111 @@
-import {BridgeBank, IbcToken, IbcToken__factory} from "../build";
-import {DependencyContainer} from "tsyringe";
+import { BridgeBank, IbcToken, IbcToken__factory } from "../build";
+import { DependencyContainer } from "tsyringe";
 import fs from "fs";
 import * as hardhat from "hardhat";
 import web3 from "web3";
 
 interface TokenAddress {
-    address: string
+  address: string;
 }
 
 async function attachIbcToken(bridgeBank: BridgeBank, token: IbcToken) {
-    return await bridgeBank.addExistingBridgeToken(token.address)
+  return await bridgeBank.addExistingBridgeToken(token.address);
 }
 
-export async function processTokenData(bridgeBank: BridgeBank, filename: string, container: DependencyContainer) {
-    const fileContents = fs.readFileSync(filename, {encoding: "utf8"})
+export async function processTokenData(
```

Comment:
Do we need some tests around process token / build token logic?

Reply:
Yes, but this is a PR for work-in-progress, so that's fine.